### PR TITLE
Fix some additional typos and improve sentence structure in a few places

### PR DESCRIPTION
**Pull request by bullywhippet**
Relates to https://github.com/cristianoliveira/awesome4girls/issues/56

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,23 +64,24 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 ### UX
   - [Ladies that UX](http://ladiesthatux.com/)
 
-    Ladies that UX is a monthly meetup that creates a welcoming, transparent community of women that work in UX, who positively promote and teach each other.
+    Ladies that UX is a monthly meet-up that creates a welcoming, transparent community of women who work in UX, who positively promote and teach each other.
```
Comment (harsh183) [inline README.md]: I think 'meetup' works.

File: README.md [hunk 2]
```diff
@@ -64,23 +64,24 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
...
   - [XX-UX](http://www.xxux.org/)
 
+    XX+UX aims to build a supportive community that fosters diversity, support, empowerment, and mentorship for women.
     The first XX+UX event was held at Google HQ in October 2013, where attendees created “story cards” that represented themselves or their journey into UX.
 
 ### General
   - [Byte Girl](http://bytegirl.com.br/) - *Loc:* <img src="https://upload.wikimedia.org/wikipedia/en/0/05/Flag_of_Brazil.svg" alt="Brazil" width="30"> - *Lang: PT-BR only*
 
-    Byte Girl is a non-profit and open event organized by volunteers focused on the female public who likes tech. It has the goal of developing women empowerment.
+    Byte Girl is a non-profit, open event organized by volunteers focused on females who likes tech. It's goal is to empower women via talks from professionals who share valuable knowledge and experiences in technology.
```
Comment (harsh183) [inline README.md]: * The comma is not needed.

* `It's` should be `Its`

File: README.md [hunk 3]
```diff
@@ -107,11 +108,11 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
...
+    The focus of Black Girls Code is to increase the number of women of color in the digital space by empowering girls of color ages 7 to 17 to become innovators in STEM fields, leaders in their communities, and builders of their own futures through exposure to computer science and technology.
 
   - [Chicas Poderosas](http://www.chicaspoderosas.org/)
 
-    Chicas Poderosas is a community working to bring Latina women in the Americas into the forefront of digital journalism. Through events, meetups and mentoring, Chicas aims to engage more women in developing news applications, producing interactive content, and integrating new tools into their newsrooms.
+    Chicas Poderosas is a community working to bring Latina women in the Americas into the forefront of digital journalism. Through events, meet-ups and mentoring, Chicas aims to engage more women in developing news applications, producing interactive content, and integrating new tools into their newsrooms.
```
Comment (harsh183) [inline README.md]: Again, I think 'meetups' works better.

File: README.md [hunk 4]
```diff
@@ -201,7 +202,7 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 
   - [SheSharp](http://www.shesharp.org/) - *Loc:* <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/5c/Flag_of_Greece.svg/30px-Flag_of_Greece.svg.png" alt="Greece" width="30">
 
-    A greek community of people dedicated to promoting women's engagement with technology & science. Its main activities consist of free workshops, meetups open to anyone interested, as well as partnerships with other tech communities and local chapters of Thessaloniki.
+    A Greek community of people dedicated to promoting women's engagement with technology & science. Its main activities consist of free workshops, meet-ups open to anyone interested, as well as partnerships with other tech communities and local chapters of Thessaloniki.
```
Comment (harsh183) [inline README.md]: Meetups over meet-ups. It flows better.

File: README.md [hunk 5]
```diff
@@ -4,7 +4,7 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 ----
 
 ## Meetups/Events
-  This section contains all specifics meetups/events.
+  This section contains all specifics meet-ups/events.
```
Comment (harsh183) [inline README.md]: Meetups over meet-ups

**Comment by harsh183** [comment]
And let me know if I missed anything or you disagree with something I said.

File: README.md (hunk 2 above)
Comment (bullywhippet) [reply README.md]: I believe that a comma works between coordinate adjectives, but I fixed the its

**Comment by cristianoliveira** [comment]
Hey guys! Sorry I went out for awhile. Let me take a quick look, but if you @harsh183 have already looked I think it won't have anything to change

**Comment by cristianoliveira** [comment]
Looks great! Merging...

Thanks @bullywhippet and @harsh183 !

**Pull request merged**